src/components/Dialog.tsx

```diff
@@ -331,11 +335,8 @@ export default function Dialog({ color, children, onClose, forceContain }: Dialo
       <ThemeProvider>
         <PopoverBoundaryProvider value={popoverRef.current} updateTrigger={updatePopover}>
           <div ref={popoverRef}>
-            <FullScreenWrapper enabled={pageCentered} onClick={onClose}>
-              <HiddenWrapper
-                constrain={pageCentered}
-                hidden={context.options?.animationType === DialogAnimationType.SLIDE}
-              >
+            <FullScreenWrapper enabled={pageCentered} onClick={pageCentered ? onClose : () => null}>
```

Comment:
I think this should continue to just be `onClose` and if you need a boolean behavior switch it should happen in that handler (or a wrapper, if that's a more applicable implementation)

Reply:
updated